docs/manual/scala/releases/Migration14.md

```diff
@@ -10,7 +10,22 @@ The version of Lagom can be updated by editing the `project/plugins.sbt` file, a
 addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.4.0-M3")
 ```
 
-## Binding services
+## API and configuration changes
+
+### Backward-incompatible API changes
```

Comment:
If I look at the MiMa config, there are a _lot_ more incompatible changes than what's listed here. I'm wondering if there's really any point enumerating them all here. The compiler will tell people about any they need to care about :smile:

I think it's only useful if we are telling people what they need to do to migrate, which in these cases is pretty trivially inferred from the errors they would get. What do you think?